eras/shelley/test-suite/bench/BenchValidation.hs

```diff
@@ -82,10 +79,11 @@ validateInput ::
   , GetLedgerView era
   , MinLEDGER_STS era
   , EraGovernance era
+  , Environment (CHAIN era) ~ ()
```

Comment:
@lehins I haven't looked at why these are necessary yet. I'm following up on this.